### ISSUE TITLE
Add Learning Rate Scheduling

### ISSUE THREAD

**Issue by Marko-Shevchuk** [PNU-SE-AIteam/cifar-10-stable-diffusion-detection#89]
Add Learning Rate Scheduling, sanity check ReduceLROnPlateau usage and patience
[Docs for LearningRateScheduling](https://keras.io/api/callbacks/learning_rate_scheduler/)
[Docs for ReduceLROnPlateau](https://keras.io/api/callbacks/reduce_lr_on_plateau/)
